package.json

```diff
@@ -50,11 +50,11 @@
     "angular-summernote": "0.8.1",
     "anythingslider": "https://github.com/CSS-Tricks/AnythingSlider/archive/v1.9.4.tar.gz",
     "backbone": "1.3.3",
-    "bootstrap": "3.3.4",
+    "bootstrap": "3.3.7",
     "bootstrap-rtl": "3.3.4",
     "chart.js": "2.1.3",
     "checklist-model": "0.10.0",
-    "ckeditor": "4.7.0",
+    "ckeditor": "4.11.1",
```

Comment:
This looks possible. We'll just need to test the 1 part of openemr that uses this:
https://github.com/openemr/openemr/blob/master/library/custom_template/custom_template.php#L53

Reply:
ckeditor is testing well :) now gonna test the moment stuff.